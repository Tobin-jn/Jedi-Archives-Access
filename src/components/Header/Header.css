header h1 {
  text-align: center;
  position: relative;
  padding: 1rem 0;
  margin: 0;
}

header {
  background: #192973;
  margin: 0;
}

.favorites {
  display: flex;
  position: absolute;
  right: 20px;
  top: 20px;
}

.favorites h3 {
  padding-right: 8px;
}

.favorites h3,
.favorites p {
  margin: 0;
  line-height: 30px;
}

.favorites p {
  text-align: center;
}

.favorites-box {
  height: 30px;
  width: 30px;
  background: #000;
  color: #fff;
}