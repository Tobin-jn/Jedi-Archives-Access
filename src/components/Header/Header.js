import React from 'react';
import './Header.css';
import PropTypes from 'prop-types';

const Header = (props) => {
  return (
    <header>
    <div className='favorites'>
      <h3>Favorites</h3>
      <div className='favorites-box'>
        <p>{props.totalFavorites}</p>
      </div>
    </div>
    </header>
  );
}

Header.propTypes = {
  // totalFavorites: PropTypes.number.isRequired
};

export default Header;