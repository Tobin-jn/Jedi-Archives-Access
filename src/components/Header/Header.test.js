import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header totalFavorites={3}/>)
  })

  it('should exist', () => {
    expect(wrapper).toBeDefined();
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})