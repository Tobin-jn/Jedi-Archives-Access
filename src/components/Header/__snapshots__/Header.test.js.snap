// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header>
  <div
    className="favorites"
  >
    <h3>
      Favorites
    </h3>
    <div
      className="favorites-box"
    >
      <p>
        3
      </p>
    </div>
  </div>
</header>
`;
